dev-java/javassist/javassist-3.29.0.ebuild

```diff
@@ -0,0 +1,156 @@
+#		at javassist.compiler.CodeGen.atStmnt(CodeGen.java:360)
+#		at javassist.compiler.ast.Stmnt.accept(Stmnt.java:53)
+#		at javassist.compiler.CodeGen.atStmnt(CodeGen.java:381)
+#		at javassist.compiler.ast.Stmnt.accept(Stmnt.java:53)
+#		at javassist.compiler.Javac.compileStmnt(Javac.java:578)
+#		at javassist.CtBehavior.insertBefore(CtBehavior.java:786)
+#		... 30 more
+#	3) testNewArray(javassist.JvstTest2)
+#	javassist.CannotCompileException: [source error] no such field: i
+#		at javassist.expr.NewArray.replace(NewArray.java:182)
+#		at javassist.JvstTest2$4.edit(JvstTest2.java:749)
+#		at javassist.expr.ExprEditor.loopBody(ExprEditor.java:238)
+#		at javassist.expr.ExprEditor.doit(ExprEditor.java:96)
+#		at javassist.CtBehavior.instrument(CtBehavior.java:744)
+#		at javassist.JvstTest2.testNewArray(JvstTest2.java:760)
+#		... 27 trimmed
+#	Caused by: compile error: no such field: i
+#		at javassist.compiler.TypeChecker.fieldAccess(TypeChecker.java:873)
+#		at javassist.compiler.TypeChecker.atFieldAssign(TypeChecker.java:296)
+#		at javassist.compiler.JvstTypeChecker.atFieldAssign(JvstTypeChecker.java:95)
+#		at javassist.compiler.TypeChecker.atAssignExpr(TypeChecker.java:255)
+#		at javassist.compiler.ast.AssignExpr.accept(AssignExpr.java:43)
+#		at javassist.compiler.CodeGen.doTypeCheck(CodeGen.java:266)
+#		at javassist.compiler.CodeGen.atStmnt(CodeGen.java:360)
+#		at javassist.compiler.ast.Stmnt.accept(Stmnt.java:53)
+#		at javassist.compiler.CodeGen.atStmnt(CodeGen.java:381)
+#		at javassist.compiler.ast.Stmnt.accept(Stmnt.java:53)
+#		at javassist.compiler.Javac.compileStmnt(Javac.java:578)
+#		at javassist.expr.NewArray.replace2(NewArray.java:252)
+#		at javassist.expr.NewArray.replace(NewArray.java:180)
+#		... 33 more
+#	4) testURL(javassist.JvstTest2)
+#	junit.framework.AssertionFailedError
+#		at junit.framework.Assert.fail(Assert.java:55)
+#		at junit.framework.Assert.assertTrue(Assert.java:22)
+#		at junit.framework.Assert.assertTrue(Assert.java:31)
+#		at junit.framework.TestCase.assertTrue(TestCase.java:200)
+#		at javassist.JvstTest2.testURL(JvstTest2.java:336)
+#	5) testMethodParameters(javassist.JvstTest4)
+#	junit.framework.ComparisonFailure: expected:<[i]> but was:<[arg0]>
+#		at junit.framework.Assert.assertEquals(Assert.java:100)
+#		at junit.framework.Assert.assertEquals(Assert.java:107)
+#		at junit.framework.TestCase.assertEquals(TestCase.java:260)
+#		at javassist.JvstTest4.testMethodParameters(JvstTest4.java:1010)
+#	6) testLocalVarAttribute(javassist.bytecode.BytecodeTest)
+#	junit.framework.AssertionFailedError
+#		at junit.framework.Assert.fail(Assert.java:55)
+#		at junit.framework.Assert.assertTrue(Assert.java:22)
+#		at junit.framework.Assert.assertTrue(Assert.java:31)
+#		at junit.framework.TestCase.assertTrue(TestCase.java:200)
+#		at javassist.bytecode.BytecodeTest.testLocalVarAttribute(BytecodeTest.java:335)
+#
+#	FAILURES!!!
+#	Tests run: 431,  Failures: 6
```

Comment:
Given that we strive to keep sizes in the ::gentoo tree minimal, as every uses sync this, I find this comment a little bit to verbose, even though I always in favor of code comments. You can definitely place it in the commit messages (as those are not typically synced to the user) and just leave a short comment that you experienced test failures even though with "mvn test" they all pass.

Reply:
I'll move it to a bug report.  That's easiest to find.

Reply:
Even better, just make sure that there is a short comment in the ebuild linking the bug.

Reply:
> Even better, just make sure that there is a short comment in the ebuild linking the bug.

Done: https://github.com/gentoo/gentoo/pull/26079/files#diff-52f551d46819df9f1e51d739c7f2b0570c32d371ee5ecb469ac768d3667e891bR25